repos/fdbt-site/src/pages/additionalPricingStructures.tsx

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable jsx-a11y/no-onchange */
```

Comment:
Can we change this to the specific line please?

Reply:
removed it was from when i was trying to use a dropdown instead